client/modules/settings/components/General.js

```diff
@@ -70,8 +70,15 @@ const General = () =>
         </Col>
         <Col lg={3}>
           <RFFieldGroup
-            name="EIN"
-            label="EIN"
+            name="receiptFields.name"
```

Comment:
The idea was you'd be able to add/remove multiple of these fields using a [redux-form FieldArray](https://redux-form.com/6.0.0-alpha.7/examples/fieldarrays/). There are other examples of its use in

> create donation items fields and household section on the customer application.

Reply:
Ok, I am working on this, but I thing I it would take me a little of time.

Can you also suggest me a good simple example or starter guia about the Redux-React combination? I need to check this as an first time user because I have not get this concept yet.

Thanks.

Reply:
Yeah that's ok, there's a lot to learn around that though.

I thought this (and the second part) was great https://egghead.io/courses/getting-started-with-redux

And for functional programming I think the first parts of this are pretty nice http://reactivex.io/learnrx/